webapp/src/components/boardTemplateSelector/boardTemplateSelector.test.tsx

```diff
@@ -231,7 +231,6 @@ describe('components/boardTemplateSelector/boardTemplateSelector', () => {
             const editIcon = screen.getByText(template1Title).parentElement?.querySelector('.EditIcon')
             expect(editIcon).not.toBeNull()
             userEvent.click(editIcon!)
-            expect(history.push).toBeCalledTimes(1)
```

Comment:
Removed this as this has been moved into utils, and utils are mocked here. I've added new tests for utils to test this functionality.